src/linalg.cu

```diff
@@ -498,16 +658,18 @@ BFstatus bfMatMul_ab_exec(BFlinalg    handle,
                           BFdtype     c_type,
                           long        c_stride,
                           long        c_batchstride) {
-	// TODO: Use batched algos here where possible
+	// We prefer the CI4@CF32 -> CF32 code
+        // second choice would be the batched algorithms
 	
 	//char* use_bf_cgemm_str = getenv("BF_CGEMM");
 	//bool use_bf_cgemm = use_bf_cgemm_str && atoi(use_bf_cgemm_str);
+       // std::cout << "nbatch: "<<nbatch<<std::endl;
 	if( //use_bf_cgemm &&
-	    n <= 12 &&
 	    trans_a == CUBLAS_OP_T && trans_b == CUBLAS_OP_N &&
 	    (a_type == BF_DTYPE_CI4  || a_type == BF_DTYPE_CI8) &&
 	    (b_type == BF_DTYPE_CI16 || b_type == BF_DTYPE_CF16 || b_type == BF_DTYPE_CF32) &&
 	    c_type == BF_DTYPE_CF32 ) {
+                std::cout<<"custom kernel"<<endl;
```

Comment:
Looks like there are few debugging-style comments/print statements here.

Reply:
whoops. Should be fixed